src/telliot_feed_examples/sources/gas_price_oracle.py

```diff
@@ -0,0 +1,113 @@
+from dataclasses import dataclass
+from dataclasses import field
+from typing import Dict
+from typing import Optional
+
+import requests
+from requests import JSONDecodeError
+from requests import Response
+from requests.adapters import HTTPAdapter
+from telliot_core.datasource import DataSource
+from telliot_core.dtypes.datapoint import DataPoint
+from telliot_core.dtypes.datapoint import datetime_now_utc
+from urllib3.util import Retry
+
+from telliot_feed_examples.utils.log import get_logger
+
+
+logger = get_logger(__name__)
+retry_strategy = Retry(
+    total=3,
+    backoff_factor=1,
+    status_forcelist=[429, 500, 502, 503, 504],
+    allowed_methods=["POST"],
```

Comment:
The allowed methods in your retry strat are wrong, needs to be "GET"

Reply:
thanks, good catch